Shared/DataTransferObjects/AttendanceForCreationDto.cs

```diff
@@ -0,0 +1,7 @@
+namespace Shared.DataTransferObjects;
+
+public record AttendanceForCreationDto(
```

Comment:
To ensure consistent validation across the application, consider adding a corresponding FluentValidation validator for this DTO or the command that uses it.

```suggestion
public record AttendanceForCreationDto(Guid EmployeeId, DateTime CheckInTime, AttendanceStatus Status, string? Notes);
```